src/util/__tests__/fields-test.js

```diff
@@ -100,8 +100,8 @@ describe('util/fields/getFieldsAsString()', () => {
                     FIELD_IS_EXTERNALLY_OWNED,
                     FIELD_ITEM_COLLECTION,
                     FIELD_CREATED_BY,
-                    FIELD_OWNED_BY,
                     FIELD_DESCRIPTION,
+                    FIELD_OWNED_BY,
```

Comment:
revert?

Reply:
Was alphabetizing, but it's probably better to do that all in one pass. Same as above